package.json

```diff
@@ -69,6 +69,7 @@
     "moment": "^2.10.3",
     "should": "^8.1.1",
     "sinon": "^1.15.4",
+    "tin": "seegno/tin#0.0.1",
```

Comment:
This dependency hasn't been published to `npm` yet due to package name collision.

Note that this will fetch the module using [this](https://github.com/npm/npm/blob/94929cf05854886b1aa9b611ad3bbb6dbbb575f3/lib/cache/add-remote-git.js#L65) algorithm.